libsolidity/ast/Types.h

```diff
@@ -396,6 +396,9 @@ class FixedPointType: public Type
 	/// smallest value in general.
 	bigint minIntegerValue() const;
 
+	/// Returns the representation of this type as an integer type. Units are 10 ^ (-fractionalDigits())
```

Comment:
Perhaps better: return smallest integer type that can hold this type with fractional parts shifted to integers, or something like hat.